proof_of_stake/src/types.rs

```diff
@@ -141,7 +140,7 @@ pub type ValidatorSlashes = NestedMap<Address, Slashes>;
 pub type EpochedSlashes = crate::epoched::NestedEpoched<
     ValidatorSlashes,
     crate::epoched::OffsetUnbondingLen,
-    crate::epoched::OffsetSlashProcessingLen,
+    crate::epoched::OffsetSlashProcessingLen, /* TODO: should this be slash procesing + cubic window? */
```

Comment:
yeah, we're reading the slashing in cubic window in `fn compute_cubic_slash_rate`

Reply:
ah, it already is that - we're using `params.slash_processing_epoch_offset`

Reply:
I rm'd the comment